astropy/units/tests/test_units.py

```diff
@@ -623,7 +623,7 @@ def test_fractional_powers():
     assert x.powers[0].denominator == 7
 
     x = u.cm ** Fraction(1, 2) * u.cm ** Fraction(2, 3)
-    assert type(x.powers[0]) == Fraction
+    assert isinstance(x.powers[0], Fraction)
```

Comment:
Looking at the context, I see your point. I think really either is fine, so maybe simplest is to just leave your change to `isinstance`.